### PR TITLE
JWWEB-443: Content-Security-Policy headers on nginx

### PR DESCRIPTION
**Pull request by stcherenkov**
- Dynamic inline scripts are external
- Build script updates `settings.js`
- Nginx adds CSP headers according to our current network activity

### CODE REVIEW

File: nginx.conf [hunk 1]
```diff
@@ -37,6 +37,13 @@ http {
         listen 80 default_server;
 
         location / {
+            set $CSP 'default-src \'self\';';
+            set $CSP '${CSP} img-src \'self\' www.google-analytics.com;';
+            set $CSP '${CSP} font-src \'self\' fonts.gstatic.com;';
+            set $CSP '${CSP} style-src \'self\' \'unsafe-inline\' fonts.googleapis.com;';
+            set $CSP '${CSP} script-src \'self\' \'sha256-dumdnmFtmXukp90IByXXb9z1Q4gMA2R+DyJAVrVqxmw=\' www.google-analytics.com;';
+            set $CSP '${CSP} connect-src \'self\' *.jwallet.network api.etherscan.io www.google-analytics.com;';
```
Comment (IViol) [inline nginx.conf]: api.etherscan.io -> *.etherscan.io

Comment (stcherenkov) [reply nginx.conf]: Fixed

File: nginx.conf [hunk 2]
```diff
@@ -37,6 +37,13 @@ http {
         listen 80 default_server;
 
         location / {
+            set $CSP 'default-src \'self\';';
+            set $CSP '${CSP} img-src \'self\' www.google-analytics.com;';
+            set $CSP '${CSP} font-src \'self\' fonts.gstatic.com;';
+            set $CSP '${CSP} style-src \'self\' \'unsafe-inline\' fonts.googleapis.com;';
+            set $CSP '${CSP} script-src \'self\' \'sha256-dumdnmFtmXukp90IByXXb9z1Q4gMA2R+DyJAVrVqxmw=\' www.google-analytics.com;';
+            set $CSP '${CSP} connect-src \'self\' *.jwallet.network *.etherscan.io www.google-analytics.com;';
+            add_header Content-Security-Policy $CSP;
```
Comment (soar) [inline nginx.conf]: Setting this option here will break running Jwallet on localhost

Comment (stcherenkov) [reply nginx.conf]: Discussed and solved in Slack

**Pull request merged**